lib/app/frontend/rapidDevServer/RapidApi.js

```diff
@@ -29,8 +29,9 @@ class RapidApi {
 
         this.setIds(sessionId, deviceId)
 
-        logger.plain(`    STACK: ${this.settings.url}`)
-        logger.plain(`    SHOP:  ${this.settings.shopNumber.replace('shop_', '')}`)
+        logger.plain(`\n    STACK:        ${this.settings.url}`)
+        logger.plain(`    SHOP:         ${this.settings.shopNumber.replace('shop_', '')}`)
```

Comment:
the prefix is not only 'shop' anymore. i'd suggest to remove the replace function here

Reply:
Done